python/graph_analyzer.py

```diff
@@ -4,15 +4,17 @@
 from itertools import chain, repeat
 from inspect import cleandoc
 from typing import List, Tuple
+
 try:
     import networkx as nx
 except ImportError as import_error:
-    sys.stderr.write((
-        '\n'
-        'NOTE: Please install networkx to be able to use graph_analyzer '
-        'module. Using Python:\n'
-        + sys.version +
-        '\n'))
+    sys.stderr.write(
+        (
+            "\n"
+            "NOTE: Please install networkx to be able to use graph_analyzer "
+            "module. Using Python:\n" + sys.version + "\n"
+        )
+    )
```

Comment:
No need to use newline, jsut use triple qoutes
```suggestion
    sys.stderr.write(
        f"""
            NOTE: Please install networkx to be able to use graph_analyzer
            module. Using Python: {sys.version }
        """
        )
    )
```